images/svc-base/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN rm -f ~/.gitconfig && \
     git config --global --add safe.directory . && \
     git lfs install && \
     GIT_LFS_SKIP_SMUDGE=1 && \
+    sudo npm install typescript -g && \
     yarn install --frozen-lockfile --production --network-concurrency 1 && \
     yarn cache clean --all
```

Comment:
should we not _remove_ typescript eventually?
(since the image is not multi-stage with a builder, need to cleanup to keep things small...)